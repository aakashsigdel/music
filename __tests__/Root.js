import 'react-native'
import React from 'react'
import Root from '../src'

import renderer from 'react-test-renderer'

it('should render <Root /> correctly', () => {
  const tree = renderer.create(
    <Root />
  ).toJSON()

  expect(tree).toMatchSnapshot()
})
