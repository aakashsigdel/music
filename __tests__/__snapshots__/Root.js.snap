exports[`test should render <Root /> correctly 1`] = `<View />`;
